### ISSUE TITLE
Add social sharing buttons

### ISSUE THREAD

**Issue by arpansac** [commudle/commudle-ng#27]
Add the links to share on social media for

- Community
- Event
- Session by experts (should appear on hover)

Probably you can make a separate component and do it

Post some example screenshots from other web apps and where you want to place these so that we can finalize the way it looks

This is going to be an important feature!

**Comment by aparth33** [comment]
I am interested.

**Comment by arpansac** [comment]
@aparth33 assigned to you!

**Comment by aparth33** [comment]
https://www.searchenginejournal.com/places-to-share-your-content/234905/

Check this site on these mentioned sites it would be great to share the content.
And also the handles I think this site has is also good.It has handles on the left side of page and it remains on the sidebar as we drag along at any given point of time.

**Comment by aparth33** [comment]
![1](https://user-images.githubusercontent.com/47942134/85156409-a9551800-b277-11ea-9c3d-2439faa3247d.JPG)

![2](https://user-images.githubusercontent.com/47942134/85156424-af4af900-b277-11ea-9b12-c8026edad302.JPG)

We can also keep the sharing handles at the end of our posts or pages.

**Comment by arpansac** [comment]
I think we can go ahead with this, you'll need to create a shared component inside 'shared-components' folder and you can pass on the links to that component. Whichever link is there, that particular link will appear.

Let's begin with the event public and event admin pages

Need any help, feel free to reach!

**Comment by arpansac** [comment]
**ROUTINECHECK**
Can you please share if you've made any progress?

**Comment by arpansac** [comment]
Do reply in the next 2 days to confirm if you're doing this!

**Comment by aparth33** [comment]
Yeah Sir I am doing this and have implemented certain things as well. You were supposed to keep a meeting to discuss things and also view the updated work.

**Comment by arpansac** [comment]
Great Parth, I suppose the programme was officially over, plus the students were having exams. Let's keep a meeting this week.

**Comment by amanintech** [comment]
I suspect a problem with social sharing.
As our app is a CSR hence the social media platform won't actually read any content from the site and won't unfurl or create a twitter-like card. That could also be a problem with the search engine as also can't read anything on the page either 
https://search.google.com/test/rich-results?id=J5gcbgwQGyxsLi5dnP82tQ
![image](https://user-images.githubusercontent.com/28403318/87458197-3eed8880-c627-11ea-8d90-2d3eb6d2b61a.png)

**Comment by arpansac** [comment]
Any progress @aparth33 ?
Let's do this a little faster, been a long time now.

**Comment by aparth33** [comment]
Yeah sure sir ! I will present to you shortly.

**Comment by aparth33** [comment]
Sir can we keep a meeting to discuss position of sharing logos on the website. Also need some help for fitting into the code.

**Comment by arpansac** [comment]
Oh, I thought you were no longer working on this...
Let's discuss this over a call?

**Comment by aparth33** [comment]
Yeah Sure Sir! Anytime tomorrow whenever you are free !

**Comment by arpansac** [comment]
Auto Alert: Removing assignment due to inactivity. Please comment again if you are working on this.

**Comment by aparth33** [comment]
Yeah sir I did and wanted to discuss with you therefore asked for time in previous comment for same .

**Comment by arpansac** [comment]
Sure, you have my number! Please feel free to ping/call :)
